addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+params [["_vehicle",0]];
+if !(_vehicle isEqualType objNull) exitWith { ERROR(format ["Vehicle must be passed to getVehicleRadios. %1 was passed instead.",_vehicle]) };
```

Comment:
`ERROR(format ["Vehicle must be passed to getVehicleRadios. %1 was passed instead.",_vehicle])`
won't compile right.
Use:
`ERROR_1("Vehicle must be passed to getVehicleRadios. %1 was passed instead.",_vehicle)`